src/components/Button/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import polished from "polished";
+
+const Button = styled.a`
```

Comment:
I think we should not to use anchor as a button. Please change component's name or html tag.